ats/anomaly_detectors/base.py

```diff
@@ -34,3 +35,58 @@ def apply(self, data, *args, **kwargs):
         """
         raise NotImplementedError()
 
+class TimeseriaAnomalyDetector(AnomalyDetector):
+    """
+    Base class for anomaly detectors wrapped from the timeseria library.
+    """  
+    
+    model_class = None
+
+    def __init__(self, *args, **kwargs):
+        if self.model_class is None:
+            raise NotImplementedError('Subclasses must define a timeseria model')
+        self.model = self.model_class(*args, **kwargs)
+        self.apply_params = {}
```

Comment:
Here you could also check if there is an `apply_params` key in the `kwargs` dict and if so you can use it (e.g. `self.apply_params = kwargs.get('apply_params', {})`